packages/office-ui-fabric-react/src/components/Persona/Persona.scss

```diff
@@ -69,26 +69,23 @@ $ms-Persona-presenceSizeXl: 28px;
   line-height: 1;
   position: relative;
```

Comment:
We do.  ms-Persona-presence is still positioned absolute, so we need relative on the parent container